camayoc/tests/qcs/test_scan.py

```diff
@@ -26,7 +26,7 @@ def profiles():
     """Gather profiles from config file."""
     try:
         profs = config.get_config()['qcs']['profiles']
-    except ConfigFileNotFoundError:
+    except (ConfigFileNotFoundError, KeyError):
```

Comment:
This fixes an issue on generating the docs. I still need to check if additional changes are needed.